src/plotting.jl

```diff
@@ -0,0 +1,28 @@
+
+interval_markers(x::AbstractInterval) = interval_markers(inclusivity(x))
+interval_markers(x::Inclusivity) = interval_markers(first(x), last(x))
+interval_markers(start::Bool, stop::Bool) = interval_markers.([start, stop])
+interval_markers(x::Bool) = x ? :vline : :none
+
+@recipe function f(xs::AbstractVector{<:AbstractInterval{T}}, ys) where T
+    new_xs = T[]
+    new_ys = []
+    markers = Symbol[]
+    for (x, y) in zip(xs, ys)
+        # To cause line to not be connected, need to add a breaker point with
+        # NaN in one of the coordinates. We put that in `new_ys` as that is probably a
+        # float. where-as new_xs is potentially a DateTime etc that would not accept NaN
+        append!(new_xs, [first(x), last(x), last(x)])
+        append!(new_ys, [y, y, NaN])
```

Comment:
What happens if you plot `DateTime` here?

Reply:
for `y`?
It errors.

But also: when would one be plotting Intervals  vs DateTimes ?

Reply:
That is fair. We should make an issue for this when this is merged

Reply:
Can we pre-make this issue?

Reply:
https://github.com/invenia/Intervals.jl/issues/86